pandas/core/arrays/numeric.py

```diff
@@ -130,3 +158,16 @@ def _arith_method(self, other, op):
             )
 
         return self._maybe_mask_result(result, mask, other, op_name)
+
+    def _apply(self, func: Callable, **kwargs) -> "NumericArray":
```

Comment:
cc @jbrockmendel @jorisvandenbossche 

shall we make this more general? (e.g. on base.py)